src/components/Navbar/index.tsx

```diff
@@ -39,8 +40,9 @@ const Navbar = () => {
         <>
             <nav className="bg-gray-900 p-4 h-[8vh]">
                 <div className="flex items-center justify-between">
-                    <Link href="/" className="text-white text-2xl font-bold">
-                        URL Shortener
+                    <Link href="/" className="flex">
+                        <Image src="/rds.png" alt="logo" width={30} height={30} className="mr-2 w-auto" />
```

Comment:
The file name can be different I think, since it's not RDS logo? 'rdsli.png' maybe?

Reply:
Done